app/views/account/admin/companies/_form.html.slim

```diff
@@ -0,0 +1,34 @@
+= simple_form_for [:account, :admin, @company] do |f|
+  .form-group.company-form
+    = f.input :name, label: t('companies.form.name_label'),
+                     placeholder: "#{t('companies.form.name_label')}",
```

Comment:
you don't need interpolation here, and bellow 
`placeholder: t('companies.form.name_label'),` should work 
and in other files too...